spec/features/step_by_step_actions_spec.rb

```diff
@@ -32,6 +32,25 @@
     end
   end
 
+  context "Step by step has been claimed for 2i" do
+    scenario "show the relevant actions to the step by step author" do
+      given_there_is_a_step_by_step_that_has_been_claimed_for_2i
+      and_I_am_the_step_by_step_author
+      when_I_visit_the_step_by_step_page
+      then_there_should_be_no_primary_action
+      and_the_secondary_action_should_be "Preview"
+      and_there_should_be_tertiary_actions_to %w(Delete)
+    end
+    scenario "show the relevant actions to the step by step 2i reviewer" do
+      given_there_is_a_step_by_step_that_has_been_claimed_for_2i
+      and_I_am_a_2i_reviewer
+      when_I_visit_the_step_by_step_page
+      then_there_should_be_no_primary_action
+      and_the_secondary_action_should_be "Preview"
+      and_there_should_be_tertiary_actions_to %w(Delete)
+    end
+  end
```

Comment:
This context block looks like it goes with https://trello.com/c/2hmErFyt/75-2i-reviewer-can-approve-and-request-changes-on-step-by-steps, rather than https://trello.com/c/Fwo562zl/72-2i-reviewer-can-claim-sbs-for-review - I think what you probably want to do is add another `scenario` to the `context "Step by step has been submitted for 2i" do` above, which tests that the "Claim for 2i review" button appears for 2i reviewers.

If you don't mind, I've copied your good work into the branch I'm working on for the approve/request-changes Trello card! Then I suggest removing it from here and working on tests for the "Claim for 2i review" instead.

Reply:
We now have [a test for the "Claim for 2i review" button](https://github.com/alphagov/collections-publisher/pull/775/files#diff-d9c1f0d36c990af71c680d57441f58eeR38). I think it's valuable to [test the actions are updated after claiming](https://github.com/alphagov/collections-publisher/pull/775/files#diff-d9c1f0d36c990af71c680d57441f58eeR44) even if they will be updated in #780.